packages/sql-clickhouse/src/ClickhouseClient.ts

```diff
@@ -231,28 +243,36 @@ export const make = (
           readonly values: Clickhouse.InsertValues<Readable, T>
           readonly format?: Clickhouse.DataFormat
         }) {
-          return FiberRef.getWith(currentQueryId, (queryId_) =>
```

Comment:
Wasn't sure about this nesting here so pls lmk if there's a prettier pattern to follow.